Gemfile

```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem "fabrication"
   gem "faker"
   gem "pry"
+  gem 'thin'
```

Comment:
I can take this out if you think it's not needed, just seemed like I needed to run this app securely for the CMS locally.